schema/bom-1.5.xsd

```diff
@@ -49,7 +61,7 @@ limitations under the License.
                 See https://cyclonedx.org/capabilities/bomlink/
             </xs:documentation>
         </xs:annotation>
-        <xs:restriction base="xs:string">
+        <xs:restriction base="xs:anyURI">
             <!-- part of the pattern is based on `bom.serialNumber`'s pattern -->
             <xs:pattern value="urn:cdx:([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})|(\{[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\})/[1-9][0-9]*"/>
```

Comment:
❌ oh, need to fix the refgex!

Reply:
Streamlined to the things used in JSON. 
removed the capability if the strange MS UUID thing with the `{}` around